api/libs/cloning/src/job-output.ts

```diff
@@ -0,0 +1,9 @@
+import { ClonePiece } from './domain';
+
+export interface JobOutput {
```

Comment:
do you anticipate `JobOutput` to potentially diverge from `JobInput`? as they stand, the former could extend the latter

Reply:
@hotzevzl although they are structurally similar, they are serving different purposes - it would be like telling one animal of different kind is extending each other. As they are structurally similar, TS will allows using them as is (with just adding one property and `...`spreading the input.

So, I wouldn't connect them just to save a few lines, as they shall remain independent.

Reply:
Yes, that was the point of my question, not to save duplication but to check if we should consider them two faces of the same coin. If you consider this not to be the case, then we're ok.

Reply:
@hotzevzl we could in this very case but it is a small internal change to the module which would need to be changed as soon as those two start to differ 🤷 Could be done if you see any benefit.